wfe/wfe.go

```diff
@@ -364,6 +344,17 @@ func (wfe *WebFrontEndImpl) Handler() http.Handler {
 	return m
 }
 
+func (wfe *WebFrontEndImpl) ManagementHandler() http.Handler {
```

Comment:
Can you add a brief godoc comment to this function that explains its purpose?

Reply:
I added the godoc comment.